docs/content/users/extend/customizing-images.md

```diff
@@ -115,6 +115,9 @@ The in-container home directory is rebuilt on `ddev restart` so if you have some
 ```Dockerfile
 RUN npm install --global spidergram
 USER $username
+# This is just an example of creating something in the home directory
```

Comment:
Trade “just” for more “something” clarity (for both people that may not have seen the `touch` command):

```suggestion
# This is an example of creating a file in the home directory
```

Reply:
Can't believe I dared to say "just"